src/main/java/com/alkemy/ong/dto/SlidePublicOrganizationDto.java

```diff
@@ -0,0 +1,12 @@
+package com.alkemy.ong.dto;
+
+import lombok.Data;
+
+@Data
+public class SlidePublicOrganizationDto {
+    private String id;
```

Comment:
same comment as above

Reply:
Ticket explicits that all fields must be included.

Reply:
Got it!